xarray/core/datatree.py

```diff
@@ -1060,6 +1060,50 @@ def drop_nodes(
         result._replace_node(children=children_to_keep)
         return result
 
+    def move(self, origin: str, destination: str, parents: bool = False) -> DataTree:
```

Comment:
There's another possible way of writing this method: `dt.move(destination)` where it's assumed that the `origin` is the current node, and the returned result is the root of the new tree.

But I think your way is probably the less surprising of the two.

Reply:
I see. Should we do both, or would that be too confusing?

Reply:
I think your way is fine! I just wanted to comment to point this out for posterity :)